test/test_jit.py

```diff
@@ -1022,10 +1018,12 @@ def foo(a, b):
             return a * b / (a - b) + b
         V = Variable
         a, b = V(torch.rand(1)), V(torch.rand(1))
-        ge = torch._C.GraphExecutor(foo, (a, b))
+        ge, _ = torch.jit.get_trace_graph(foo, (a, b))
+        m = torch.jit.ScriptModule()
+        m._create_method_from_graph("forward", ge.graph())
```

Comment:
I'm changing this so we don't extract the graph and build a new module. Do you know if there are other places we do this?

Reply:
It's also done in assertExportImport